judge/views/contests.py

```diff
@@ -130,10 +130,16 @@ class ContestMixin(object):
     slug_url_kwarg = 'contest'
 
     @cached_property
-    def is_organizer(self):
+    def is_editor(self):
         if not self.request.user.is_authenticated:
             return False
-        return self.object.organizers.filter(id=self.request.profile.id).exists()
+        return self.request.profile.id in self.object.editor_ids
+
+    @cached_property
+    def is_tester(self):
+        if not self.request.user.is_authenticated:
+            return False
+        return self.request.profile.id in self.object.tester_ids
```

Comment:
Just curious, why do we use `profile.id in contest.tester_ids` here but `contest.testers.filter(id=user.profile.id).exists()` is `contest.is_accessible_by`?

Reply:
Consistency hard.